core/components/atoms/textarea/__stories__/variants/Disable.story.jsx

```diff
@@ -1,22 +1,16 @@
 import * as React from 'react';
 import { Text, Textarea } from '@/index';
 import { action } from '@/utils/action';
+import '../style.css';
 
 export const Disable = () => {
-  const innerStyle = {
-    display: 'flex',
-    'flex-direction': 'column',
-    alignItems: 'center',
-    maxWidth: '300px',
-  };
-
   const BooleanValue = [true, false];
 
   return (
     <div className="Row">
       {BooleanValue.map((value, ind) => (
-        <div style={innerStyle} className="mr-9 mb-5" key={ind}>
-          <div style={innerStyle} className="mr-9 mb-5">
+        <div className="mr-9 mb-5 d-flex flex-column align-items-center Textarea-wrapper" key={ind}>
```

Comment:
Please refactor it in a cleaner manner. We can reduce the extra html and classnames by using `Row`, `Column` and `Label` like this example https://codesandbox.io/s/young-bush-r9jf2c?file=/index.js